### DISCUSSION TITLE
Wanted: a one-click report spam button

### DISCUSSION THREAD

**Discussion by erwanor** [community/community#102690] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

There has been a highly noticeable uptick of scam/spam posts pinging my account, as well as a few others (e.g. screenshot 1). Unfortunately, the report flow is completely decoupled from the actual user view (e.g. screenshot 2).

It would be considerably more helpful if there was a simple "Report as spam" button to surface those to Github moderation. One can imagine that processing these spam reports could be expedited by running checks like:
- Is this account new?
- Has this account committed code?
- Are they matching a known posting patterns that other spam reports have surfaced 

etc.

<img width="519" alt="Screenshot 2024-01-29 at 5 05 13 PM" src="https://github.com/community/community/assets/7871622/e0947a2c-5067-4c0d-aa51-5a355e90a91e">

<img width="237" alt="Screenshot 2024-01-29 at 5 07 09 PM" src="https://github.com/community/community/assets/7871622/043f32cc-a09f-4f5d-a3bd-c47a0d2827df">

**Comment by AleixMT** [comment]
This seems like a super useful thing to have in GitHub. I was in need for it many times in the past.

**Comment by armujahid** [comment]
Another thing I noticed is that spam report option is only visible in github mobile app. I couldn't find spam report option in web app.

**Comment by WaS-Studio** [comment]
its unfortunate, as its only been very recently that it's become a big issue. They're probably reviewing more robust solutions to address this emergence.

**Comment by crrusucosmin** [comment]
Hi! My GitHub account rusucosmin got disabled after I fell into the trap of this spam message - I need back access to the account, or I would like to get all my private repositories so I can move then somewhere else
